### ISSUE TITLE
Decouple PEtab and AMICI

### ISSUE THREAD

**Issue by dilpath** [ICB-DCM/pyPESTO#1301]
Currently, the PEtab importer depends on AMICI, so I have to install AMICI to use it. However, I use PEtab+pyPESTO in a few applications that do not involve AMICI. Using the PEtab importer to create the pyPESTO problem is convenient, and I supply a custom (non-AMICI) objective to `PetabImporter.create_problem(objective=custom_objective)`. Having it default to using an AMICI objective is fine, but then the imports etc need to handled properly when AMICI is not available.

**Comment by PaulJonasJost** [comment]
Agreed. Will need to adjust this for me as well for integrating roadrunner. Will have a look at tit.
